CONTRIBUTING.md

```
@@ -0,0 +1,320 @@
+
+To create the bundle, you will need the 'yarn' package management tool to be installed. Then, you can run the following commands on a *nix system to download the required packages and build the bundle:
+
+```bash
+$ cd $PGADMIN4_SRC
+$ make install-node
+$ make bundle
+```
+
+On Windows systems (where "make" is not available), the following commands can be used:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)